src/Identifier/PasswordIdentifier.php

```diff
@@ -51,6 +51,8 @@ class PasswordIdentifier extends AbstractIdentifier
      * - `fields` The fields to use to identify a user by:
      *   - `username`: one or many username fields.
      *   - `password`: password field.
+     * - `forceLowercase` Forces identifier to lowercase when comparing to
```

Comment:
This name feels scary and I wouldn't use it. Could we do `forceLowercaseUsername` or `lowercaseUsername` instead?

Reply:
My thoughts exactly, `lowercaseUsername` sounds much better.